app/javascript/components/profile/contributions-list/AuthoringContributionsList.tsx

```diff
@@ -1,26 +1,75 @@
 import React from 'react'
-import { ExerciseWidget } from '../../common'
+import { ExerciseWidget, Pagination } from '../../common'
 import { ExerciseAuthorship } from '../../types'
+import { FetchingBoundary } from '../../FetchingBoundary'
+import { ResultsZone } from '../../ResultsZone'
+import { useIsMounted } from 'use-is-mounted'
+import { useList } from '../../../hooks/use-list'
+import { usePaginatedRequestQuery, Request } from '../../../hooks/request-query'
+
+type PaginatedResult = {
+  results: readonly ExerciseAuthorship[]
+  meta: {
+    currentPage: number
+    totalCount: number
+    totalPages: number
+  }
+}
+
+const DEFAULT_ERROR = new Error('Unable to load contributions')
```

Comment:
Maybe make this specific?

```suggestion
const DEFAULT_ERROR = new Error('Unable to load authoring contributions')
```